packages/grain_dypgen/grain_dypgen.0.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+maintainer: "https://github.com/ocaml/opam-repository/issues"
+authors: ["Emmanuel Onzo" "Philip Blair"]
+homepage: "https://github.com/grain-lang/dypgen"
+license: "CeCILL-B"
+build: make
+depends: [
+  "ocaml" {>= "4.07.0"}
+  "ocamlfind"
```

Comment:
```suggestion
  "ocamlfind" {build}
```
From a brief look at the code, I think you only use this for the build. After this I think we can merge, sorry for not spotting it earlier

Reply:
No worries; I've updated the PR.